scripts/build/libc/glibc.sh

```diff
@@ -136,11 +137,24 @@ do_libc_backend() {
 
         CT_mkdir_pushd "${CT_BUILD_DIR}/build-libc-${libc_mode}${extra_dir//\//_}"
 
+        target=${CT_TARGET}
+        case "${target}" in
+            # SPARC quirk: glibc 2.23 and newer dropped support for SPARCv8 and
+            # earlier (corresponding pthread barrier code is missing). Until this
+            # support is reintroduced, configure as sparcv9.
+            sparc-*)
+                if [ "${CT_LIBC_GLIBC_2_23_or_later}" = y ]; then
+                    target=${target/#sparc-/sparcv9-}
+                fi
+                ;;
+        esac
+
```

Comment:
I thought about it, but it is really glibc quirks with regard to the target tuple (we'll have one more as a part on multilib branch I am preparing). That is, there is nothing wrong with spart-*-* target per se, it is *glibc* that does not implement some crucial parts on sparcv8.